pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TransactionEndToEndTest.java

```diff
@@ -96,6 +96,8 @@ public class TransactionEndToEndTest extends TransactionTestBase {
     protected static final String TOPIC_OUTPUT = NAMESPACE1 + "/output";
     protected static final String TOPIC_MESSAGE_ACK_TEST = NAMESPACE1 + "/message-ack-test";
     protected static final int NUM_PARTITIONS = 16;
+    private static final int waitTimeForCanReceiveMSgInSec = 5;
+    private static final int waitTimeForCannotReceiveMSgInSec = 1;
```

Comment:
may be also should 5, to avoid the incorrect verify?

Reply:
> may be also should 5, to avoid the incorrect verify?

+1

Reply:
Done